packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -238,10 +347,62 @@ class _ValidationDetailScreen extends StatelessWidget {
             ' and Custom Schemes in your app. Web check are done for the web association'
             ' file on your website. App checks are done for the intent filters in'
             ' the manifest and info.plist file, routing issues, URL format, etc.',
-            style: Theme.of(context).textTheme.bodySmall,
+            style: textTheme.bodyMedium!.copyWith(
+              color: colorScheme.onSurface.withOpacity(0.6),
```

Comment:
can we use `theme.subtleTextStyle` here instead? We should be using existing DevTools theme components as much as possible so that the deep link tool matches the surrounding DevTools app